test/StringUtils.test.cpp

```diff
@@ -116,60 +129,83 @@ TEST(String, stringFrom)
 	EXPECT_EQ("129"s, NAS2D::stringFrom<unsigned char>(129));
 	EXPECT_EQ("-1"s, NAS2D::stringFrom<signed char>(-1));
 
-	EXPECT_EQ("0"s, NAS2D::stringFrom<unsigned short>(0));
-	EXPECT_EQ("1"s, NAS2D::stringFrom<unsigned short>(1));
-
-	EXPECT_EQ("0"s, NAS2D::stringFrom<unsigned int>(0));
-	EXPECT_EQ("1"s, NAS2D::stringFrom<unsigned int>(1));
-
-	EXPECT_EQ("0"s, NAS2D::stringFrom<unsigned long>(0));
-	EXPECT_EQ("1"s, NAS2D::stringFrom<unsigned long>(1));
-
-	EXPECT_EQ("0"s, NAS2D::stringFrom<unsigned long long>(0));
-	EXPECT_EQ("1"s, NAS2D::stringFrom<unsigned long long>(1));
-
-	EXPECT_EQ("-1"s, NAS2D::stringFrom<short>(-1));
-	EXPECT_EQ("0"s, NAS2D::stringFrom<short>(0));
-	EXPECT_EQ("1"s, NAS2D::stringFrom<short>(1));
-
-	EXPECT_EQ("-1"s, NAS2D::stringFrom<int>(-1));
-	EXPECT_EQ("0"s, NAS2D::stringFrom<int>(0));
-	EXPECT_EQ("1"s, NAS2D::stringFrom<int>(1));
-
-	EXPECT_EQ("-1"s, NAS2D::stringFrom<long>(-1));
-	EXPECT_EQ("0"s, NAS2D::stringFrom<long>(0));
-	EXPECT_EQ("1"s, NAS2D::stringFrom<long>(1));
-
-	EXPECT_EQ("-1"s, NAS2D::stringFrom<long long>(-1));
-	EXPECT_EQ("0"s, NAS2D::stringFrom<long long>(0));
-	EXPECT_EQ("1"s, NAS2D::stringFrom<long long>(1));
-
-	EXPECT_EQ("-1.000000"s, NAS2D::stringFrom<float>(-1));
-	EXPECT_EQ("-1.000000"s, NAS2D::stringFrom<float>(-1.0));
-	EXPECT_EQ("-1.100000"s, NAS2D::stringFrom<float>(-1.1f));
-	EXPECT_EQ("0.000000"s, NAS2D::stringFrom<float>(0));
-	EXPECT_EQ("0.000000"s, NAS2D::stringFrom<float>(0.0));
-	EXPECT_EQ("1.000000"s, NAS2D::stringFrom<float>(1));
-	EXPECT_EQ("1.000000"s, NAS2D::stringFrom<float>(1.0));
-	EXPECT_EQ("1.100000"s, NAS2D::stringFrom<float>(1.1f));
-
-	EXPECT_EQ("-1.000000"s, NAS2D::stringFrom<double>(-1));
-	EXPECT_EQ("-1.000000"s, NAS2D::stringFrom<double>(-1.0));
-	EXPECT_EQ("-1.100000"s, NAS2D::stringFrom<double>(-1.1));
-	EXPECT_EQ("0.000000"s, NAS2D::stringFrom<double>(0));
-	EXPECT_EQ("0.000000"s, NAS2D::stringFrom<double>(0.0));
-	EXPECT_EQ("1.000000"s, NAS2D::stringFrom<double>(1));
-	EXPECT_EQ("1.000000"s, NAS2D::stringFrom<double>(1.0));
-	EXPECT_EQ("1.100000"s, NAS2D::stringFrom<double>(1.1));
-
-	EXPECT_EQ("-1.000000"s, NAS2D::stringFrom<long double>(-1));
-	EXPECT_EQ("-1.000000"s, NAS2D::stringFrom<long double>(-1.0));
-	EXPECT_EQ("-1.100000"s, NAS2D::stringFrom<long double>(-1.1L));
-	EXPECT_EQ("0.000000"s, NAS2D::stringFrom<long double>(0));
-	EXPECT_EQ("0.000000"s, NAS2D::stringFrom<long double>(0.0));
-	EXPECT_EQ("1.000000"s, NAS2D::stringFrom<long double>(1));
-	EXPECT_EQ("1.000000"s, NAS2D::stringFrom<long double>(1.0));
-	EXPECT_EQ("1.100000"s, NAS2D::stringFrom<long double>(1.1L));
+}
+
+
+template<typename T>
+class StringFromSignedIntegerTest : public ::testing::Test
+{
+  public:
+	  StringFromSignedIntegerTest<T>()
+	{}
```

Comment:
This probably don't need to be explicit. If it does, it can probably be `= default;`